test/runtests.jl

```diff
@@ -21,7 +21,7 @@ for T in (Float32, Float64)
             # Simple math #
             ###############
             t = rand(Tensor{order, dim, T})
-            t_one = one(Tensor{order, dim, T})
+            t_ones = ones(Tensor{order, dim, T})
```

Comment:
But it should probably be used or tested somehow :p
Can rewrite constructor tests in a separate PR